README.md

```diff
@@ -169,18 +169,35 @@ fnm env --use-on-cd | source
 ```
 
 #### PowerShell
-
-Add the following to the end of your profile file:
-
+Create your PowerShell Profile File (if it doesn't exist):
 ```powershell
-fnm env --use-on-cd | Out-String | Invoke-Expression
+# Check if the directory for the profile exists, if not, create it
+$profileDir = Split-Path $profile
+if (-Not (Test-Path -Path $profileDir)) {
+    New-Item -ItemType Directory -Path $profileDir -Force
+}
+
+# Create the profile file if it does not exist
+if (-Not (Test-Path -Path $profile)) {
+    New-Item -ItemType File -Path $profile -Force
+}
 ```
 
-- For macOS/Linux, the profile is located at `~/.config/powershell/Microsoft.PowerShell_profile.ps1`
-- On Windows to edit your profile you can run this in a PowerShell
+Open the Profile File in a text editor:
+- On Windows to edit your profile you can run this in PowerShell
   ```powershell
   notepad $profile
   ```
+- For macOS/Linux, the profile is located at `~/.config/powershell/Microsoft.PowerShell_profile.ps1`
```

Comment:
I think we can avoid this and simply have the following as "edit your profile" code:

```powershell
New-Item -ItemType Directory -Force -Path (Split-Path -Parent $profile)
notepad $profile
```

Reply:
I don't know, if you tested this then I would presume that it is better. My advice would be to test it in a virtual machine with a brand new PowerShell 5 or 7 to check if this works.

Reply:
@Schniz Have you look at this?